packages/react/kibana_context/common/theme.ts

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the "Elastic License
+ * 2.0", the "GNU Affero General Public License v3.0 only", and the "Server Side
+ * Public License v 1"; you may not use this file except in compliance with, at
+ * your election, the "Elastic License 2.0", the "GNU Affero General Public
+ * License v3.0 only", or the "Server Side Public License, v 1".
+ */
+
+import { EuiThemeSystem, EuiThemeAmsterdam } from '@elastic/eui';
+
+export interface ThemeConfig {
```

Comment:
There are discussions whether to make separate out themes and color modes. This config object allows for such changes and future expansion of what a theme configures